documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -876,14 +875,14 @@ When you have only a few local changes, the easiest way would be to cherry-pick
 % git checkout main
 % git cherry-pick old_branch..your_branch
 ....
-Or alternatively, do the same thing with rebase:
+Alternatively, do the same thing with rebase:
 [source,shell]
 ....
 % git rebase --onto main master your_branch
```

Comment:
Should we edit the `master` part of this example?

Reply:
No. This whole section can be deleted to. It's about migrating from the old github mirror, which is 3 years gone now. So the example with master is right for that, but it's no longer relevant, imho, so it should just be deleted rather than updated and/or fixed.... While one might still have work in a git-svn tree, I think that time has passed as well...

Reply:
Resolved with deletion of the entire subsection in [`f1c95fe` (#182)](https://github.com/freebsd/freebsd-doc/pull/182/commits/f1c95fe2485e42b6b8b085ba8d280a5d27328610).